### ISSUE TITLE
Got wired inference results when running examples in Tutorial. Thanks!

### ISSUE THREAD

**Issue by pangxiaohai** [tensorflow/nmt#250]
I run examples shown in [Neural Machine Translation (seq2seq) Tutorial](https://www.tensorflow.org/tutorials/seq2seq) , but inference results looks wired. And train stage looks OK. 

I use python2.7 and tensorflow 1.4. I use NMT codes on master branch.

Train cmd I use is `python -m nmt.nmt \
    --src=en --tgt=vi \
    --vocab_prefix=/tmp/nmt_data/vocab  \
    --train_prefix=/tmp/nmt_data/train \
    --dev_prefix=/tmp/nmt_data/tst2012  \
    --test_prefix=/tmp/nmt_data/tst2013 \
    --out_dir=/tmp/nmt_model \
    --num_train_steps=12000 \
    --steps_per_stats=100 \
    --num_layers=2 \
    --num_units=128 \
    --dropout=0.2 \
    --metrics=bleu`

Then I run following cmd `python -m nmt.nmt \
    --out_dir=/tmp/nmt_model \
    --inference_input_file=/tmp/my_infer_file.en \
    --inference_output_file=/tmp/nmt_model/output_infer
`
but got a wired inference result. Is there anything wrong? Has anyone else met same issue? Thanks!

The inference results is like: 

`Và là là là là là là là là là là là là là là là là là là là là là là là là là là là là .
Và là là là là .
Và là là là là là là là là là là là là là là là là là là là là là là là là là là là là là là là là là là là là .
Và là là là là là là là là là là là là là là là là là là là là là là là là .
Và là là là là là là là là là là là là là là là là là là là là là là là .
Và là là là là là là là là là là là là là .
Và là là là là là là là là là là là là là là là là là .
Và là là là là là là là là là là là là là là là là là là là là là là là là là là là là là .
Và là là là là là là là là là là là là là là là là là là là .
Và là là là .
Và là là là là là là là là là là là là là .
Và là là là là là là là là là là là là là là là là là là là là là là là .
Và là là là là là là là là là là là là là là là là là là là là là là là .
Và là là là là là là là là là là là là là là là là là .
Và là là là là là là là là .
Và là là là là là là là là là là là là là là là là là là là là là là là là .
Và là là là là là là là là là là là là là là là là là là là là là là là là là là là là là là là là là .
Và là là là là là là là là là là .
Và là là là là là là là là là là là là .
Và là là là là là là là là là là là là là là là là là là .
Và là là là là là là là là là là là là là là là là là là là là .
Và là là .
`
